int32 state
int32 pickup_zone
float32 pickup_zone_x
float32 pickup_zone_y
